easybuild/easyconfigs/a/AmrPlusPlus/AmrPlusPlus-2.0-GCC-8.3.0.eb

```diff
@@ -0,0 +1,74 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	MIT
+#
+# Notes::
+##
+
+easyblock = 'CmdCp'
+
+name = 'AmrPlusPlus'
+version = '2.0'
+
+github_account = 'meglab-metagenomics'
+
+homepage = 'https://megares.meglab.org/amrplusplus/latest/html/v2'
+description = """AmrPlusPlus v2.0 can process the raw data from the sequencer,
+identify the fragments of DNA, and count them. It also provides a count of the
+polymorphisms that occur in each DNA fragment with respect to the reference database.
+"""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/%(github_account)s/%(name)s_v%(version_major)s/archive']
+sources = ['8afc0af5234f8e322cc0f5e40cc553ecd5c932b8.zip']
+checksums = ['04af67517b752e15b5add27f4584f8c11293a0ed1353d3c1dd60c3c6e2dcb329']
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('BWA', '0.7.17'),
+    ('Trimmomatic', '0.39', '', ('Java', '11')),
+    ('SAMtools', '1.10'),
+    ('Nextflow', '19.12.0', '', True),
+]
+
+skipsteps = ['configure', 'build']
```

Comment:
@robqiao Good that it builds now, but it is a bit strange - more like a `binary`.
Any idea why the build was failing>

Reply:
Changing to `Binary` easyblock makes sense I think, which would allow removing this `skipsteps`...

Reply:
@verdurin it looks like this package is a simple script pipeline and `make` step prompts error due to missing `make.in`